recipes/_configure_server.rb

```diff
@@ -249,11 +249,11 @@
 
   license_pools['pools'].each do |pool, pool_config|
     pool_max_size = CernerSplunk.convert_to_bytes pool_config['size']
-    server_stanzas["lmpool:#{pool}"] = {
+    server_stanzas["lmpool:#{node.run_state['type']}"] = {
```

Comment:
Won't this value be empty because we are evaluating this at the compile time? Can you show what the server.conf looks like on the license server?

Reply:
Also, are we grouping pools by license type?

Reply:
Talking with Charlie, he mentioned that for the fixed-sourcetype license the folder and pool are the same, which is fixed-sourcetype_Uppercase(sha256(sourcetypeName)). So, I was trying to account for that here